.drone.yml

```diff
@@ -398,6 +398,36 @@ steps:
     RUST_BACKTRACE: 1
   commands:
     - cargo test --release --workspace
+- name: run-unit-tests-multibackend-rockdb
+  image: tezedge/tezedge-ci-builder:latest
+  pull: if-not-exists
+  user: root
+  environment:
+    SODIUM_USE_PKG_CONFIG: 1
+    RUST_BACKTRACE: 1
+  commands:
+    - cd storage
+    - cargo test --release --features maindb-backend-rocksdb
+- name: run-unit-tests-multibackend-sled
+  image: tezedge/tezedge-ci-builder:latest
+  pull: if-not-exists
+  user: root
+  environment:
+    SODIUM_USE_PKG_CONFIG: 1
+    RUST_BACKTRACE: 1
+  commands:
+    - cd storage
+    - cargo test --release --features maindb-backend-sled
+- name: run-unit-tests-commit-log
+  image: tezedge/tezedge-ci-builder:latest
+  pull: if-not-exists
+  user: root
+  environment:
+    SODIUM_USE_PKG_CONFIG: 1
+    RUST_BACKTRACE: 1
+  commands:
+    - cd storage
+    - cargo test --lib commit_log -- --nocapture
```

Comment:
@mambisi 
please add here also `--release`  because all before was with `--release`  so this last one, does not need to compile/download all deps for debug

Reply:
Ok